src/commands/help.ts

```diff
@@ -47,6 +47,7 @@ ${b('\\  /\\')}  ${m('/\\  /')}   ${b('/\\  /')}
 
 export class HelpCommand implements Command {
   name = 'help';
+  aliases = ['h'];
```

Comment:
lets actually remove this for now, the original request was just for an `install` alias. I'd rather tackle all others together when we decide on a rule for which aliases to support overall.